### PR TITLE
Add pronouns to faculty staff profile

### PR DESCRIPTION
**Pull request by silviot**
See #461

### CODE REVIEW

File: src/isaw.facultycv/isaw/facultycv/browser/templates/faculty_detail.pt [hunk 1]
```diff
@@ -30,6 +30,8 @@
                 </div>
                 <div class="ProfileInfo">
                     <h1 class="ProfileName" tal:content="context/Title">Name</h1>
+                    <div class="ProfilePronouns" tal:condition="context/getPronouns|nothing"
+                         tal:content="structure context/getPronouns" />
```
Comment (jessesnyder) [inline src/isaw.facultycv/isaw/facultycv/browser/templates/faculty_detail.pt:34]: I'm guessing you're doing something deliberate by using `structure` here, even though the field accessor does not return HTML?

**Review by jessesnyder — APPROVED** [review]
Just one specific question on the template. 

Looks like a straightforward solution!

File: src/isaw.facultycv/isaw/facultycv/browser/templates/faculty_detail.pt (hunk 1 above)
Comment (silviot) [reply src/isaw.facultycv/isaw/facultycv/browser/templates/faculty_detail.pt:34]: Nope! It was an oversight. I copy pasted from a field that does output html. Thanks for pointing it out! People should be able to use the pronoun `<they>` if they want! Our code should not stop them ;-)

Comment (jessesnyder) [reply src/isaw.facultycv/isaw/facultycv/browser/templates/faculty_detail.pt]: Glad to be able to avert that injustice!

**Pull request merged**